sass/_clients.scss

```diff
@@ -1,77 +1,161 @@
-.featured-clients {
+.hero-header {
+    text-align: center;
+    margin-top: 90px;
+    margin-bottom: 81px;
+    color: #000000;
+    width: 100%;
+    max-width: 651px;
+
+
+    @media (max-width: 767px) {
+        max-width: 312px;
+        margin-top: 56px;
+        margin-bottom: 34px;
+    }
+
+    h1 {
+        width: 100%;
+        font-style: normal;
+        font-weight: 700;
+        font-size: 40px;
+        line-height: 120%;
+    }
+
+    p {
+        width: 100%;
+        font-style: normal;
+        font-weight: 400;
+        font-size: 18px;
+        line-height: 120%;
+    }
+}
+
+section {
     display: flex;
-    gap: .5rem;
-    overflow-x: auto;
+    flex-direction: column;
+    align-items: center;
+}
+
+
+.featured-clients {
+    flex-grow: 3;
+    width: 100%;
+    display: grid;
+    grid-auto-flow: column;
+    grid-template-columns: 1fr 1fr 1fr;
+
+
+    @media (max-width: 767px) {
+        width: unset;
+        grid-template-columns: 1fr;
+        grid-auto-flow: row;
+    }
```

Comment:
I think it would make sense to use a flexbox + max-width to get the responsiveness "for free" instead of a forcing the layout on one or three columns